messenger_test.go

```diff
@@ -54,8 +56,9 @@ func (s *MessengerSuite) SetupTest() {
 
 	config := whisper.DefaultConfig
 	config.MinimumAcceptedPOW = 0
-	s.shh = whisper.New(&config)
-	s.Require().NoError(s.shh.Start(nil))
+	shh := whisper.New(&config)
+	s.shh = gethbridge.NewGethWhisperWrapper(shh)
+	s.Require().NoError(shh.Start(nil))
```

Comment:
I don't see a special reason to do it. Starting a node is implementation-specific, or not?

Reply:
I don't think so. Assume one wants to create the bridge and pass a struct around but not necessarily start Whisper yet. With this setup, it means both the wrapper and `shh` need to be kept in that struct.

Reply:
Maybe I'm not picturing what you mean correctly. Would that mean that we'd also wrap the `p2p.Server` struct that is used as argument, to be used later on when the bridge would start the node? The goal here is to avoid leaking those types from this API.

Reply:
My idea is as simple as having a possibility to do:
```go
s.shh.Start()
```

But I forgot about `p2p.Server` as an argument... That would require to pass `p2p.Server` to `whisper.New` constructor so that we don't need to wrap `p2p.Server`. However, that's even more complex change so let's keep it as it is.